client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -173,6 +176,16 @@ public ReaderGroupConfigBuilder startFromCheckpoint(final Checkpoint checkpoint)
            return this;
        }
 
+       /**
+        * Ensure not more than maximum outstanding checkpoint requests are allowed at any given time.
+        * @param maxOutstandingCheckpointRequest Max allowed outstanding checkpoint request.
+        * @return Reader group config builder.
+        */
+       public ReaderGroupConfigBuilder maxPendingCheckpoints(final int maxOutstandingCheckpointRequest) {
```

Comment:
This method is not actually needed. If you delete it it will be auto generated.

Reply:
Yes... The lombok annotation @Builder will auto generate all the simple setter methods. You only need to specify the ones that override normal behavior.

Reply:
(done)